### PR TITLE
[web] Replace react-linkify with markdown

### PR DESCRIPTION
**Pull request by KatPo**

### CODE REVIEW

File: web/markdown/markdown.react.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ class Markdown extends React.PureComponent<Props> {
   }
 
   render() {
-    return <div>{this.output(this.ast)}</div>;
+    return this.output(this.ast);
```
Comment (Ashoat) [inline web/markdown/markdown.react.js:30]: Good call!

**Comment by KatPo** [comment]
Do we want do add markdown to `Entry` in web? We're not using linkify there, and you didn't mention it in the comment in #37, so I didn't do that yet

File: web/markdown/markdown.react.js [hunk 2]
```diff
@@ -4,14 +4,16 @@ import * as React from 'react';
...
+import { rules, advancedRules } from './rules.react';
 
 type Props = {|
   children: string,
+  onlyBasicRules?: boolean,
 |};
 class Markdown extends React.PureComponent<Props> {
   static propTypes = {
     children: PropTypes.string.isRequired,
+    onlyBasicRules: PropTypes.bool,
```
Comment (Ashoat) [inline web/markdown/markdown.react.js:16]: Let's pass the `rules` function in here instead of this boolean

File: web/markdown/rules.react.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import * as React from 'react';
 
 import { urlRegex } from 'lib/shared/markdown';
 
-export default function rules() {
+export function rules() {
```
Comment (Ashoat) [inline web/markdown/rules.react.js:8]: Let's call this `linkRules`

File: web/markdown/rules.react.js [hunk 4]
```diff
@@ -36,3 +36,11 @@ export default function rules() {
     },
   };
 }
+
+// function will contain additional rules for message formatting
+export function advancedRules() {
```
Comment (Ashoat) [inline web/markdown/rules.react.js:41]: And call this `markdownRules`. Also use a single `export { linkRules, markdownRules };` block at the bottom

File: web/markdown/markdown.react.js [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
-import rules from './rules.react';
-
 type Props = {|
   children: string,
+  rules: MarkdownRules,
 |};
 class Markdown extends React.PureComponent<Props> {
   static propTypes = {
     children: PropTypes.string.isRequired,
+    onlyBasicRules: PropTypes.bool,
```
Comment (Ashoat) [inline web/markdown/markdown.react.js:16]: Still need to update the `PropType`

**Comment by Ashoat** [comment]
Awesome!!

**Pull request merged**